### ISSUE TITLE
(Bug report) "Add to homescreen" on Android sets wrong url if Trilium is run in sub-directory via reverse proxy

### ISSUE THREAD

**Issue by nimonimonimo** [zadam/trilium#4702]
### Trilium Version

0.62.4

### What operating system are you using?

Other (specify below)

### What is your setup?

Local + server sync

### Operating System Version

FreeBSD 13.2

### Description

Hi,

Thanks for an amazing app.

I have been using Trilium for several months now and likes it very much. I run it on my FreeBSD server behind a reverse proxy via nginx. I run it in a sub-directory similar to the example in the documentation:
```  
location /trilium/ {
    proxy_pass http://127.0.0.1:8080/;
    proxy_http_version 1.1;
    proxy_set_header Upgrade $http_upgrade;
    proxy_set_header Connection 'upgrade';
    proxy_set_header Host $host;
    proxy_cache_bypass $http_upgrade;
} 
```
I have no problem connecting to it -- either from client on linux or mac os laptop -- as well as via webbrowser on any platform.

My only problem is that when I from my Android phone wants to add it as a shortcut via the browser menu: "Add to homescreen" it will be added with url consisting of only the domain name without the sub-directory. I believe this is because the generated "manifest.webmanifest"-file which looks like this:
```
{
  "name": "Trilium Notes",
  "short_name": "Trilium",
  "description": "Trilium Notes is a hierarchical note taking application with focus on building large personal knowledge bases.",
  "theme_color": "#333333",
  "background_color": "#1F1F1F",
  "display": "standalone",
  "scope": "/",
  "start_url": "/",
  "icons": [
    {
      "src": "/assets/vX/images/app-icons/ios/apple-touch-icon.png",
      "sizes": "180x180",
      "type": "image/png"
    },
    {
      "src": "/assets/vX/images/app-icons/png/512x512.png",
      "sizes": "512x512",
      "type": "image/png"
    }
  ]
```

I suspect that "scope" or "start_url" should include "/trilium". 

Would it be possible to add support for this?

Thanks in advance!

**Comment by Nriver** [comment]
Samsung browser works fine. This should be an issue with your browser app.

**Comment by Nriver** [comment]
I believe I've identified the issue. It seems to be related to the Android launcher. Currently, I'm using a third-party launcher called "Nova Launcher," and the shortcuts function smoothly with both the Samsung browser and Chrome browser. However, when I switch back to the default launcher on my phone, the shortcuts behave as you described.

So, you might want to change your launcher app now.

**Comment by JIeJaitt** [comment]
hi, Can Trilium be deployed on freebsd systems?

**Comment by nimonimonimo** [comment]
> @nimonimonimo hi, Can Trilium be deployed on freebsd systems?

Yes, wasn't any larger problems except for that I needed to install some packages. My install notes looks like this:
`
pkg install "nasm",
        "screen",
        "libtool",
        "autoconf",
        "libxkbfile",
        "libX11",
        "yarn-node18",
        "npm-node18",
        "vim",
        "python",
        "git",
	"py311-fail2ban",
        "py311-certbot",
        "py311-supervisor"


sysrc -f /etc/rc.conf supervisord_enable="YES"

pw user add -n trilium  -c "Trilium Notes" -m

cd /tmp
#fetch https://github.com/tagattie/FreeBSD-Electron/releases/download/v27.0.2/electron27-27.0.2-freebsd13-amd64.pkg
fetch https://github.com/tagattie/FreeBSD-Electron/releases/download/v32.1.1/electron32-32.1.1-freebsd14-amd64.pkg
# check exact version and install it manually


mkdir -p /opt/
cd /opt
git clone -b stable https://github.com/zadam/trilium.git

chown -R trilium: /opt/trilium

su -l trilium -c "cd /opt/trilium && npm install"
su -l trilium -c "cd /opt/trilium && npm rebuild"
su -l trilium -c "cd /opt/trilium && npm run webpack"



#Add this to /usr/local/etc/supervisord.com:
[program:trilium]
command=/usr/local/bin/start_trilium
directory=/opt/trilium
user=trilium
autostart=true
autorestart=true
stderr_logfile=/var/log/supervisor-trilium-error.log
stdout_logfile=/var/log/supervisor-trilium-stdout.log


#Create file /usr/local/bin/start_trilium with this content:
#!/bin/csh
setenv TRILIUM_DATA_DIR /storage/trilium-data
cd /opt/trilium
node src/www


chmod +x /usr/local/bin/start_trilium

service supervisord start
`
